src/room/track/RemoteVideoTrack.ts

```diff
@@ -123,6 +123,7 @@ export default class RemoteVideoTrack extends RemoteTrack {
     }
     this.elementInfos = this.elementInfos.filter((info) => info !== elementInfo);
     this.updateVisibility();
+    this.updateDimensions();
```

Comment:
I guess especially because we're calling this in a loop, it would make sense to replace this with the debounced version
```suggestion
    this.debouncedHandleResize();
```

Reply:
@lukasIO Done.